src/DataManagement/Saving/AlternativeSavingCalculator.cs

```diff
@@ -6,20 +6,20 @@
 
 namespace ePiggyWeb.DataManagement.Saving
 {
-    public static class AlternativeSavingCalculator
+    public class AlternativeSavingCalculator
     {
         private static decimal RegularSavingValue { get; } = 0.25M;
         private static decimal MaximalSavingValue { get; } = 0.5M;
         private static decimal MinimalSavingValue { get; } = 0.1M;
 
-        public static int GetSuggestedExpensesOffers(IEntryList entryList, IGoal goal, IList<ISavingSuggestion> entrySuggestions,
-            List<SavingSuggestionByMonth> monthlySuggestions, decimal startingBalance, SavingType savingType = SavingType.Regular)
+        public CalculationResults GetSuggestedExpensesOffers(IEntryList entryList, IGoal goal, decimal startingBalance, SavingType savingType = SavingType.Regular)
         {
-            entrySuggestions ??= new List<ISavingSuggestion>();
-
+            var entrySuggestions = new List<ISavingSuggestion>();
+            var monthlySuggestions = new List<SavingSuggestionByMonth>();
```

Comment:
Nice one, since no point passing list as parameter since they are empty, considering that result can be returned in class or struct which holds all the required values.

Reply:
I was slightly annoyed at this for quite some time, finally made a result class :D